app/src/sharedTest/java/org/oppia/app/topic/revision/TopicReviewFragmentTest.kt

```diff
@@ -1,4 +1,4 @@
-package org.oppia.app.topic.review
```

Comment:
@maheshpaliwal This file name is still `TopicReviewFragmentTest`, please change that to `TopicRevisionFragmentTest`